### PR TITLE
Mixed check

### PR DESCRIPTION
**Pull request by galeksandrp**
If you are not impressed by https://github.com/franciskim/check-mixed-content/commit/master
```bash
check-mixed-content --depth=1 --url {}
```
can be replaced by https://github.com/bramus/mixed-content-scan/commit/master
```bash
~/.composer/vendor/bin/mixed-content-scan https://{} --no-crawl --format=json | jq -e 'select(.level==400)'
```

### CODE REVIEW

**Comment by J0WI** [comment]
This cannot replace manual review, but it could be a help.
*ping* @Hainish

File: .travis.yml [hunk 1]
```diff
@@ -5,10 +5,4 @@ services:
 script:
   - test/travis.sh
 env:
-  - TEST=firefox FIREFOX=firefox-dev
-  - TEST=firefox FIREFOX=firefox-latest
-  - TEST=firefox FIREFOX=firefox-esr-latest
-  - TEST=chromium
-  - TEST=rules
-  - TEST=fetch
-  - TEST=preloaded
```
Comment (Hainish) [inline .travis.yml]: why?

File: src/chrome/content/rules/freerunet.ru.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!--
 www.freerunet.ru self signed
 -->
-<ruleset name="freerunet.ru" platform="mixedcontent">
+<ruleset name="freerunet.ru">
```
Comment (Hainish) [inline src/chrome/content/rules/freerunet.ru.xml]: let's keep changes granular.  this belongs in another PR

File: test/travis.sh [hunk 3]
```diff
@@ -79,6 +79,11 @@ if [ "$RULESETS_CHANGED" ]; then
     docker run --rm -ti -v $(pwd):/opt -e RULESETS_CHANGED="$RULESETS_CHANGED" node bash -c "cd /opt/utils/hsts-prune && npm install && node index.js"
     [ `git diff --name-only | wc -l` -eq 0 ]
   fi
+  
+  if [ "$TEST" == "mixed" ]; then
+    echo >&2 "Ensuring rulesets do not introduce targets containing active mixed content."
+    docker run --rm -ti -v $(pwd):/opt -e RULESETS_CHANGED="$RULESETS_CHANGED" node bash -c "cd /opt && apt-get update && apt-get install -y libxml2-utils && npm install -g galeksandrp/check-mixed-content#245e56ad5fd5412bdad986276911e5460d9e48c6 && xmllint --xpath '//ruleset[not(@platform)]/target' $RULESETS_CHANGED | grep -q -v '^$' || exit 0 && xmllint --xpath '//ruleset[not(@platform)]/target' $RULESETS_CHANGED | sed 's/<target host=\"//g' | sed 's&\"/>&\n&g' | grep -v '*' | xargs -n1 -i bash -c 'check-mixed-content --depth=1 --url {}' | grep ' has active mixed content!' && exit 1"
```
Comment (Hainish) [inline test/travis.sh]: This belongs in a `utils/` subpath, as we've done before for other tests.

**Comment by J0WI** [comment]
If it's possible to add labels depending on Travis results there would be a lot of use cases for it.
E.g. redirects that are not the same on http/https as in the rule, or when there is a big difference in the content.

**Comment by J0WI** [comment]
Maybe we can use this tool to fix #529?
